### DISCUSSION TITLE
Pages - Https not working / certificate not valid

### DISCUSSION THREAD

**Discussion by govert** [community/community#47321] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

My pages repository is https://github.com/Excel-DNA/Excel-DNA.github.io and I've set up a custom domain of excel-dna.net.
The http://excel-dna.github.io is correctly redirected to http://excel-dna.net , so I believe the DNS is set up correctly.
However, https fails with an error NET::ERR_CERT_COMMON_NAME_INVALID.
I see it shows the site is deployed to http://excel-dna.net, while for another GitHub site I here where https is working, it instead shows https://... as the location

![image](https://user-images.githubusercontent.com/414659/218662748-bee471f3-cff1-45ed-a970-df581bba0922.png)

My DNS check is successful.

![image](https://user-images.githubusercontent.com/414659/218662976-a7fb27d7-c57e-45c1-ae64-38e8eb7d73d9.png)

I set the new site up and changed the name servers and DNS entries for the domain on Sunday.
Please help to see what else might be a problem.

I did rename repositories in an attempt to make things work, but I can see that the correct site is server on the non-http site, it is just that the https certificate is not getting updated.

**Comment by govert** [comment, marked as answer]
Yay! The problem is fixed now - the site is served over https and the "Enforce HTTPS" checkbox is available.

**Reply by focat69** [reply]
bruh
